content/en/docs/architecture/release-gating.md

```diff
@@ -34,6 +36,11 @@ Poor candidates include:
 OpenShift CI creates thousands of clusters a week - we can certainly afford to run a few more tests.  However, we want to spread that investment to derive the maximum benefit for everyone, so having OpenShift CI run 30 variations of your exact same tests in slightly different configurations 4 times a day may not be the best way to test.
 {{< /alert >}}
 
+### A new approach to release gating
+Creating useful, stable and testable release gating jobs has proven to be difficult at scale.  The combinatorial explosion of cloud providers, layered products and unique configurations means most jobs have a _>1% flake rate_.  To "meet the tests where they are" the Technical Release Team has created a new type of job called the _aggregated job_.
+
+Aggregated Jobs are powerful because they allow a suite of tests to become blocking in spite of a high flake rate.  This is accomplished by analyzing the output from numerous parallel runs to see if payload quality remains as good or better than the historical mean.  If not, the next course of action is to isolate the changes and revert them.  This allows other non-regressing changes to make their way into release.  More importantly it avoids the exponentially difficult task of debugging multiple regressions in parallel (eg, it's hard to debug a layered product regression on a platform that no longer installs properly).
```

Comment:
is this the right place to officially explain how the aggregated jobs work? I was curious recently and was pointed to this [doc](https://docs.google.com/document/d/16E0dLFLbLBTe0J4fUd_55I-8bJc9t22BwsdWqFuutaQ/edit), but that seems more like a working doc. maybe there is a way to add a few bullet points like @dgoodwin gave me in this [thread](https://coreos.slack.com/archives/C01CQA76KMX/p1635959130294300).

Reply:
I feel like this page is a good place to at least introduce the direction we're moving for release gating.  I definitely feel like more detail is needed which is I think what you're asking for.  Are you OK with this merging as is and I follow up with a more permanent summary of the working doc you referred to?  One that have that PR I could replace much of the text above with a link.

Reply:
Actually, I took your suggestion and transformed the working doc you referenced into something more permanent.  I included in this PR under a new section called "release oversight".

Reply:
I like the additional docs you added. very helpful to understand what's going on. And love the new aggregated job paradigm the TRT has implemented. great work :)